lib/ui/shared/dialogs/remind_me_dialog/remind_me_dialog_viewmodel.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:hng/app/app.locator.dart';
+import 'package:hng/services/notification_service.dart';
+import 'package:intl/intl.dart';
+import 'package:stacked/stacked.dart';
+import 'package:stacked_services/stacked_services.dart';
+
+class RemindMeDialogViewModel extends BaseViewModel {
+  final navigationService = locator<NavigationService>();
+  final notificationService = locator<NotificationService>();
+
+  String twentyMinutes = DateFormat('h:mm a')
+      .format(DateTime.now().add(const Duration(seconds: 10)));
```

Comment:
You're receiving 10 seconds instead of the 20 minutes you said

Reply:
That was a mistake just to test for the video. I'll change it now